src/pudl/scripts/dbt_helper.py

```diff
@@ -20,10 +21,50 @@
 
 configure_root_logger()
 logger = get_logger(__name__)
+logger.parent.propagate = False
+
 
 ALL_TABLES = [r.name for r in PUDL_PACKAGE.resources]
 
 
+def _prettier_yaml_dumps(yaml_contents: dict[str, Any]) -> str:
+    """Dump YAML to string that Prettier likes."""
+
+    class PrettierCompatibleDumper(yaml.Dumper):
+        """Custom Dumper that indents lists like prettier does.
+
+        Default dumper behavior::
+
+            foo:
```

Comment:
Wait, was the old behavior in DbtSchema.to_yaml` like this? Because then I would have expected to see errors due to incorrect alignment.

Reply:
Yes, this was the old behavior - it's "valid YAML" but also `prettier` runs in our pre-commit hooks which fixes all the alignment issues.